src/materialized/src/bin/materialized/main.rs

```diff
@@ -51,6 +51,36 @@ fn main() {
 fn run() -> Result<(), anyhow::Error> {
     panic::set_hook(Box::new(handle_panic));
 
+    // Clear out any Timely/Differential environment variables so that we have
+    // exact control all configuration parameters to Timely/Differential.
+    //
+    // NOTE(benesch): it would be nice if Timely/Differential used a
+    // configuration struct or builder API for all parameters, rather than
+    // environment variables.
```

Comment:
💯 this.

Reply:
I don't think this comment is appropriate - file an issue/PR on timely.